### PR TITLE
Sorting Import with Isort

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
Sorting Import with isort -y -rc
see #393

### CODE REVIEW

**Comment by elelay** [comment]
I'd like `isort -c -rc` to be added to `make lint`. You've removed it from .travis.yml: did it cause problems?

File: tools/win_installer/misc/depcheck.py [hunk 1]
```diff
@@ -12,15 +12,16 @@
...
 """
 
-import subprocess
 import os
+import subprocess
 import sys
 from multiprocessing import Process, Queue
 
 import gi
-gi.require_version("GIRepository", "2.0")
```
Comment (elelay) [inline tools/win_installer/misc/depcheck.py]: This must be kept there (before from gi import GIRepository).
See http://pygobject.readthedocs.io/en/latest/guide/api/api.html#gi.require_version

File: share/gpodder/extensions/notification-win32.py [hunk 2]
```diff
@@ -44,16 +44,16 @@
 __only_for__ = 'win32'
 
 import functools
+import logging
 import os
 import os.path
 
 import gi
-gi.require_version('Gtk', '3.0')
```
Comment (elelay) [inline share/gpodder/extensions/notification-win32.py]: must be kept there

File: src/gpodder/gtkui/draw.py [hunk 3]
```diff
@@ -23,21 +23,16 @@
 #  Thomas Perl <thp@perli.net>, 2007-11-25
 #
 
-import gpodder
+import io
+import math
 
+import cairo
 import gi
-gi.require_version('PangoCairo', '1.0')
```
Comment (elelay) [inline src/gpodder/gtkui/draw.py]: must be kept there

File: src/gpodder/gtkui/main.py [hunk 4]
```diff
@@ -17,86 +17,61 @@
...
 #
 
+import cgi
+import collections
+import glob
+import logging
 import os
 import platform
-import gi
-gi.require_version('Gtk', '3.0')
```
Comment (elelay) [inline src/gpodder/gtkui/main.py]: idem

**Comment by MarkusHackspacher** [comment]
Travis show lots of errors, but on my computer work it fine and the isort version are the same.
https://travis-ci.org/MarkusHackspacher/gpodder/builds/409304093

**Pull request merged**